deploy/deploy.go

```diff
@@ -73,6 +74,8 @@ type deploySummary struct {
 	NumLocal, NumRemote, NumUploads, NumDeletes int
 }
 
+var metaMD5Hash = "md5chksum" // the meta key to store md5hash in
```

Comment:
Can make this a const, no?

Also, I don't think you need to ToLower it down below, since it's a const.